### ISSUE TITLE
curl keeps failing (I think)

### ISSUE THREAD

**Issue by naturallymitchell** [MunifTanjim/luver#4]
```
mitchell@penguin:~$ luver install 5.0.3
/tmp/luver-lua-5.0.3--xG46ylpFd ~
Warning: Failed to create the file                                                                                                                                         
Warning: /home/mitchell/.local/share/luver/.downloads/lua-5.0.3.tar.gz: No 
Warning: such file or directory
##############                                                                                                                                                          8.6%
curl: (23) Failed writing body (0 != 16384)
mitchell@penguin:~$ luver install 5.4.3
/tmp/luver-lua-5.4.3--kLS2xIckk ~
Warning: Failed to create the file                                                                                                                                         
Warning: /home/mitchell/.local/share/luver/.downloads/lua-5.4.3.tar.gz: No 
Warning: such file or directory
#######                                                                                                                                                                 4.6%
curl: (23) Failed writing body (0 != 16384)
mitchell@penguin:~$ 
```

**Comment by MunifTanjim** [comment]
Thanks for reporting. I'll fix this asap. In the meantime you can run this to fix it:

```sh
mkdir -p /home/mitchell/.local/share/luver/.downloads
```

**Comment by MunifTanjim** [comment]
This should be fixed with 7ea40c1d64c322f3ae3f0afaecd012a5bc8428a4

**Comment by naturallymitchell** [comment]
Yep, works now.  Thanks
